e2e/cephfs.go

```diff
@@ -362,6 +369,274 @@ var _ = Describe("cephfs", func() {
 					}
 				})
 
+				By("create a PVC clone and bind it to an app", func() {
+					v, err := f.ClientSet.Discovery().ServerVersion()
+					if err != nil {
+						e2elog.Logf("failed to get server version with error %v", err)
+						Fail(err.Error())
+					}
+					// snapshot beta is only supported from v1.17+
+					if v.Major > "1" || (v.Major == "1" && v.Minor >= "17") {
```

Comment:
Just check the reverse, and `Skip()` the test is the version is too low. This reduces an indention for the actual test and makes thins easier to read

Reply:
@nixpanic atm, this check is same as `rbd`. Can we change both in a separate PR?

Reply:
make sure Skip will skip only this test not the other tests below.

Reply:
@nixpanic atm, this check is same as `rbd`.  may be this check can be like this.
-`if v.Major > "1" || (v.Major == "1" && v.Minor >= "17") {
+ if  (v.Major == "1" && v.Minor >= "17")

@nixpanic wdyt?

Reply:
Marking this in tracker for common solution between cephfs and rbd. thanks .

Reply:
Please do not change the existing unrelated rbd test in this PR, that should be done separately.